flag.go

```diff
@@ -136,7 +136,10 @@ type StringSlice []string
 
 // Set appends the string value to the list of values
 func (f *StringSlice) Set(value string) error {
-	*f = append(*f, value)
+	for _, s := range strings.Split(value, ",") {
```

Comment:
Good point, the choice of `,` here is consistent with how environment variables are treated. I'm just concerned with unexpectedly breaking compatibility with use cases that depend on allowing `,`s to be specified in their values.

In hindsight, it feels like the delimiter for environment variables could also be configurable.